elements/pfe-avatar/src/pfe-avatar.scss

```diff
@@ -1,35 +1,45 @@
 @import "../../pfe-sass/pfe-sass";
 
-:host {
-  --pfe-avatar--width: 128px;
+$LOCAL: avatar;
+$LOCAL-VARIABLES: (
+  size: 128px
+);
+
+ // TODO: This will be removed at 1.0 in favor of size
```

Comment:
```suggestion
 // TODO: This will be removed at 1.0 in favor of width
```

Reply:
No, width is being applied to height and width so size is the corrected BEM naming.  This comment is correct.

Reply:
`size` is the var listed in the `backwards-compatibility` map though. So if its being removed, what var will take its place?

Reply:
That's how you set up backwards compatibility.  If you look at the compiled output: 

```
  width: var(--pfe-avatar--size, var(--pfe-avatar--width, 128px));
  height: var(--pfe-avatar--size, var(--pfe-avatar--width, 128px));
```

Size is the preferred variable, then it falls back to width if size does not exist.  The entire backwards compatibility maps will be deleted for 1.0 release leaving instead:
```
  width: var(--pfe-avatar--size, 128px);
  height: var(--pfe-avatar--size, 128px);
```

Reply:
aha, thank you :)